<launch>
  <node name="thrust_cal" pkg="jaws2_navigation" type="thrust_cal"/>
</launch>
